### ISSUE TITLE
Windows 10 does not display nirlauncher/sysinternals shortcuts in folder

### ISSUE THREAD

**Issue by rasa** [lukesampson/scoop#2124]
In Windows 7, the  nirlauncher and sysinternals packages nicely appear as folders in the "Scoop Apps" folder. On Windows 10, the folder names are ignored, resulting, and each package's entries appear intermixed with the other non-folder apps. I spent a couple of hours trying to fix this, but eventually gave up. Here are three possible solutions:

1. Instead of creating the shortcut folder under `Scoop Apps`, we create it in the same folder the `Scoop Apps` folder is created in. Elegant, but will users expect this, since other apps get installed in `Scoop Apps`. Perhaps we should simply do away with `Scoop Apps`?

2. We put the app's shortcuts in `%userprofile%\scoop\apps\app\current\shortcuts`, and create a single shortcut in `Scoop Apps` that opens an Explorer window on that directory. Kludgy.

3. Instead of creating `Nirlauncher\app` shortcuts, we create `zNirlaucher - app` shortcuts. This forces these multiple-app packages to appear at the end of the `Scoop Apps` list. Kludgy.

Feedback?

**Comment by stkb** [comment]
Personally I've never really found it useful that the shortcuts are all put in the 'Scoop Apps` folder, so I'd do away with that.

However we'd then need to be careful that the scoop installer doesn't overwrite existing (non-scoop) shortcuts.

**Comment by r15ch13** [comment]
@stkb That's why they get put in the 'Scoop Apps' folder 😁
But does it really matter? Do you really search a program in the menu manually? Typing in a programs name is much faster.

**Comment by stkb** [comment]
@r15ch13 Yeah 99% of the time I type the name too rather than browse (so I'm not *too* bothered by this issue either) but I can see it being handy in some cases, especially with these packages.

To fix the collision issue, we could add a flag (probably in the description field) to shortcuts we create. Then if a shortcut already exists without our flag, we add `" (Scoop)"` to the name of our shortcut.

**Comment by rasa** [comment]
How about we set the folder name as a config option:
```
scoop config startmenu_folder “Scoop Apps”
```
and default to “” for new installs?

**Comment by Stanzilla** [comment]
Sounds good to me. The new convention is to not use folders in the start menu anyway.

**Comment by rasa** [comment]
The Windows 10 April 2018 update hides the `Scoop Apps` folder. See https://github.com/lukesampson/scoop/issues/2233

**Comment by kodybrown** [comment]
I like the idea of a flash to indicate if I want Start Menu shortcuts added. Such as:

    scoop install -add-shortcut 7zip

**Comment by tech189** [comment]
Another way to deal with collisions would be to add a warning like we have for shims:
```
Creating shim for 'inkview'.
WARN  Overwriting shim ('inkview.exe' -> 'inkview.com') installed from inkscape
```
So something like:
```
Creating shortcut for Inkscape (inkscape.exe)
WARN  Overwriting shortcut (Inkscape -> '~\scoop\apps\inkscape\current\bin\inkscape.exe') already in Start Menu
```

**Comment by elisimpson** [comment]
Does this 

> How about we set the folder name as a config option:
> 
> ```
> scoop config startmenu_folder “Scoop Apps”
> ```
> 
> and default to “” for new installs?

Does this actually work? There's nothing is the config help that mentions it. This would be really useful as I always have to manually move the shortcuts out of the folder and they get recreated every update.

**Comment by rashil2000** [comment]
> Does this actually work? There's nothing is the config help that mentions it. This would be really useful as I always have to manually move the shortcuts out of the folder and they get recreated every update.

No, this hasn't been implemented yet
